basiclti/web-ifp/pom.xml

```diff
@@ -31,6 +31,10 @@
             <groupId>org.sakaiproject.kernel</groupId>
             <artifactId>sakai-component-manager</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
You don't **have** to include commons-lang3 with commons-lang here. commons-lang has the same [NumberUtils.toLong](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/math/NumberUtils.html#toLong(java.lang.String,%20long)) you're using here. 

Though it probably would be worth upgrading to lang3 and removing lang here

Reply:
Thanks.  I think I will just use lang and leave upgrade to lang3 later so I feel like I don't need to test as much.